resources_test_scripts/visium_tiny.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -eo pipefail
+
+# get the root of the directory
+REPO_ROOT=$(git rev-parse --show-toplevel)
+
+# ensure that the command below is run from the root of the repository
+cd "$REPO_ROOT"
+DIR="resources_test/visium"
+
+# from https://www.10xgenomics.com/resources/datasets/human-ovarian-cancer-1-standard
+mkdir -p $DIR && cd $DIR
```

Comment:
I don't really like scripts that use `cd` because it changes my workdir without me knowing. Also, using cd at the end of the script might not work when the script craches. Can you update the commands to use `$DIR` as output?>